### DISCUSSION TITLE
Help required with modularizing lunarvim config

### DISCUSSION THREAD

**Discussion by aniketgm** [LunarVim/LunarVim#3798] [Q&A] [answered]
I have modularized my config as shown below, since the configs are growing day by day,
Not sure what I'm doing wrong here ? Do I need to add/remove something ? Can someone please me help with this modularization.

![module-not-found](https://user-images.githubusercontent.com/14215698/215992068-2f90bd77-29c1-4f31-9192-ec24c6e53a7b.png)

And `colorschemes.lua` and other lua files are simply returning the table.

![module-not-found-](https://user-images.githubusercontent.com/14215698/215998734-533c1a74-39fe-40a4-844c-93c46478c872.png)

Any other suggestions are also appreciated.

**Comment by PasiBergman** [comment, marked as answer]
As answered in the LunarVim discord, you need to fix 2 things
1. "custom" is missing from the require paths, e.g. `require("custom.plugins.colorschemes")`
2. `colorschemes.lua` contains an list of plugins, which you then import **inside** of the list of plugins -> list of plugins containing multiple lists of plugins.

Use e.g. 
```lua
vim.list_extend(lvim.plugins, require("custom.plugins.colorscheme"))
vim.list_extend(lvim.plugins, require("custom.plugins.cybu"))
```
etc. to extend the plugins list. See `:help vim.list_extend` for more info.
